core/src/test/scala/org/apache/spark/ExternalShuffleServiceSuite.scala

```diff
@@ -43,6 +43,9 @@ class ExternalShuffleServiceSuite extends ShuffleSuite with BeforeAndAfterAll {
     conf.set("spark.shuffle.manager", "sort")
     conf.set("spark.shuffle.service.enabled", "true")
     conf.set("spark.shuffle.service.port", server.getPort.toString)
+
+    // local-cluster mode starts a Worker which would start its own shuffle service without this:
+    conf.set("spark.worker.shouldHostShuffleServiceIfEnabled", "false")
```

Comment:
Do we actually ever want to start an external shuffle service in a local cluster? If not I think it makes more sense to just set `spark.shuffle.service.enabled` to false in `LocalSparkCluster` (we already do this for the REST submission server for Master)